haskell/private/dependencies.bzl

```diff
@@ -85,10 +85,10 @@ def gather_dep_info(ctx):
         static_libraries_prof = [],
         dynamic_libraries = set.empty(),
         interface_dirs = set.empty(),
-        prebuilt_dependencies = set.from_list(ctx.attr.prebuilt_dependencies),
+        prebuilt_dependencies = set.empty(),
```

Comment:
Is this field still needed?

Reply:
Yes, it’s used internally to carry everything added via `haskell_import`.
When https://github.com/tweag/rules_haskell/issues/440 is tackled we should be able to get rid of this field.